src/command/feed/upload.ts

```diff
@@ -5,6 +5,8 @@ import { FeedCommand } from './feed-command'
 export class Upload extends FeedCommand implements LeafCommand {
   public readonly name = 'upload'
 
+  public readonly aliases = ['u', 'up']
```

Comment:
Maybe I would remove these command aliases since `feed` command has **up**load and **up**date in the same time.

Reply:
That's a good point, let's just remove the aliases here